database/migrations/mysql/1597442135_add_eks_encryption_config_column.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `amazon_eks_clusters` ADD COLUMN `encryption_config` JSON;
```

Comment:
Question: Do we have to save these details in the database? Is this something we want to return in details?

Reply:
I guess it's not really a use-case, no.
I took the auth config as an example that's why I included it in the database.
I'm gonna try and move the encryption config out of the EKS cluster model then into the EKS cluster directly.

Reply:
I moved the encryption config out of the cluster model [here](https://github.com/banzaicloud/pipeline/compare/d9ba632ed04d449bcfc4601cee8c5caeeea20c72..0df34bec89a6668d01dc76aac60a168d02b4ce89).

And removed the database migration files [here](https://github.com/banzaicloud/pipeline/compare/0df34bec89a6668d01dc76aac60a168d02b4ce89..194ac0de0ef9f8145ee2995cb56c716007c389c2).